test/e2e/tests/phishing-controller/phishing-detection.spec.js

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable mocha/no-hooks-for-single-case, mocha/no-sibling-hooks, no-loop-func */
```

Comment:
Could you add a brief explanation here for why these rules have been disabled? The reasons you stated below do make sense, but we can say that here in a comment. I don't want to normalize the behavior of disabling ESLint rules without explanation because people tend to do it for the wrong reasons, without understanding what the rule is for.

It might be easier to move the `no-loop-func` to the line it applies to as well, since there's only one violation. The explanation would make more sense alongside the relevant code.

Reply:
I slapped them up here because eslint rule comments are so noisy and make code harder to read (I also think many of our eslint rules themselves neccessitate writing code that is harder to read in the first place, but I digress).

I won't die on this hill, so I'll make the changes as requested.

Reply:
`no-loop-func` is a frustrating one. Generally it's a good rule in that it can catch some nasty errors, but it can't tell the difference between the problematic cases (where a reader might wrongly expect a variable to have the same value inside the function as it does when the function is declared in the loop), and the innocuous cases like this (references that aren't specific to a given iteration).

I would say something like
```
      // This rule is flagging unsafe references to `server` and `driver`, but they are being used
      // safely here. We are intentionally sharing one instance for each test.
      // eslint-disable-next-line no-loop-func
```

For the other two, it maybe suffices to say that you've broken those rules to improve readability.

Reply:
>I also think many of our eslint rules themselves neccessitate writing code that is harder to read in the first place, but I digress

Please follow up in Slack with specifics! We should have rules that encouraging writing easier to read code. Though to some degree that will be subjective. Maybe some of the things you are annoyed with can be reconsidered.

>eslint rule comments are so noisy and make code harder to read

To clarify, I don't generally object to file-wide ignore comments, I do think that would be better for the ones regarding test hooks in this case.

For the `no-loop-func` one, it seemed easier to understand that one in-context but whatever you would prefer is fine by me, as long as there's at least a one-line comment somewhere on why we're disabling it.

Reply:
Done!